playground/frontend/playground_components/lib/widgets/split_view/widget.dart

```diff
@@ -0,0 +1,199 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:provider/provider.dart';
+
+import '../../constants/sizes.dart';
+import '../../generated/assets.gen.dart';
+import 'controller.dart';
+
+class _Pan {
+  Widget child;
+  double size;
+
+  _Pan({
+    required this.child,
+    required this.size,
+  });
+}
+
+class SplitView extends StatelessWidget {
+  final SplitViewController controller;
+  final Widget firstChild;
+  final Widget secondChild;
+  final ThemeData themeData;
```

Comment:
What is the benefit of passing `themeData` here and not taking it from the tree?

Reply:
Removed.